src/utils.py

```diff
@@ -745,10 +769,14 @@ def extract_preds_and_types(
     return preds, types
 
 
-def filter_static_operators(
+def get_static_preds_atoms(
         ground_ops: Collection[GroundNSRTOrSTRIPSOperator],
-        atoms: Collection[GroundAtom]) -> List[GroundNSRTOrSTRIPSOperator]:
-    """Filter out ground operators that don't satisfy static facts."""
+        atoms: Collection[GroundAtom]
+) -> Tuple[Set[Predicate], Set[GroundAtom]]:
+    """Get predicates and atoms that are static w.r.t.
+
```

Comment:
autoformat fail! just write out "with respect to" so it doesn't think it's a new sentence?

Reply:
(i'm curious then if it gets rid of this blank line)

Reply:
ha

Reply:
it changed it to:

```
    """Get predicates and atoms that are static with respect to the
    operators."""
```

(two lines.) is this correct style? note that it's two lines, but the triple quotes are on the same line. I guess if both lint and autoformatting pass, that's the rule of law now? @ronuchit

Reply:
yeah, it's the rule of law now :) i don't understand the rules for docstring quotations but whatever